Add Grok deprecation release note (#2972)
SumoLogic/sumologic-documentation: blog-cse/2023-08-22-application-update.md

---
title: August 22, 2023 - Application Update
keywords:
  - cloud siem
  - grok
  - parsing
tags: [application update]
authors:
  - url: https://help.sumologic.com/release-notes-cse/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

import useBaseUrl from '@docusaurus/useBaseUrl';

## Deprecation Notice

> After careful evaluation, we have determined that Grok patterns are deprecated immediately for customers not using the feature in the last 30 days. A more robust and configurable solution is already available for customers in the Sumo Logic parsers. More details on how parsing works in Sumo Logic can be found in the [Parsing Language Reference Guide](/docs/cse/schema/parsing-language-reference-guide).

Reviewer: ```suggestion
After careful evaluation, we have deprecated Grok patterns immediately for customers who've not used the feature in the last 30 days. Our more robust and configurable solution is already available for customers in the Sumo Logic parsers. More details on how parsing works in Sumo Logic can be found in the [Parsing Language Reference Guide](/docs/cse/schema/parsing-language-reference-guide).
```

> For customers who are still using Grok, further communication along with a path to migration to the Sumo Logic parsers will be provided in the coming weeks.

Reviewer: ```suggestion
For customers who are still using Grok, further communication along with a path to migrate to the Sumo Logic parsers will be provided in the coming weeks.
```
